typo3/cms-core/2020-11-17-2.yaml

```diff
@@ -0,0 +1,13 @@
+title: 'TYPO3-CORE-SA-2020-010: Cross-Site Scripting in Fluid view helpers'
+link: 'https://typo3.org/security/advisory/typo3-core-sa-2020-010/'
+branches:
+    10.x:
+        time: '2020-11-17 08:51:00'
+        versions: ['>=10.0.0', '<10.4.10']
+    9.x:
+        time: '2020-11-17 08:49:54'
+        versions: ['>=9.0.0', '<9.5.23']
+    8.x:
+      time: '2020-11-17 08:55:33'
+      versions: ['>=8.7.0', '<8.7.38']
+reference: 'composer://typo3/cms-core'
```

Comment:
this looks wrong to me. the linked page says that this affects `typo3/cms-fluid`, not `typo3/cms-core`